intake/tests/models/test_form_submission.py

```diff
@@ -82,33 +80,6 @@ def test_get_transfer_action_returns_none(self):
         self.assertIsNone(
             submission.get_transfer_action(request))
 
-    def test_qualifies_for_fee_waiver_with_public_benefits(self):
```

Comment:
@bengolder Can you recall why we removed these tests? Do they refer to a defunct feature?

Reply:
I moved this functionality and this test into the submissions service.